app/Middleware/Auth.ts

```diff
@@ -0,0 +1,72 @@
+import { GuardsList } from '@ioc:Adonis/Addons/Auth'
```

Comment:
How much of this file comes by generation or is written by you?

Reply:
The support for auth is added by using **node ace configure @adonisjs/auth** and this command generates for me the files in the image below:
![image](https://user-images.githubusercontent.com/19782721/128248103-41d6634d-4769-41e5-b9aa-cfececc3379b.png)

Reply:
Then I get to modify the files I need when the need arises such as the users' migration and its model to suit what I want.

Reply:
Interessting! Nice stuff!